src/cloud/components/atoms/MarkdownView/index.tsx

```diff
@@ -297,6 +302,7 @@ const MarkdownView = ({
           onRenderRef.current()
         }
       } catch (err) {
+        console.error(err)
```

Comment:
What is the purpose of this log, the `setState({ type: 'error', err })` does not allow to see the error anywhere?